src/client/directory.ts

```diff
@@ -1,3 +1,5 @@
+import {Intent} from './internal';
```

Comment:
This is moved out as we want to use it in `Model` and `AppDirectory`, but if we export it directly from here, we'll get name collisions from `main.ts`'s wildcard imports.